src/newSeed/stage1.ts

```diff
@@ -13,6 +14,25 @@ export class Stage1 extends BaseStage {
     this.seedConfig.general.customLinks.splice(index, 1);
   }
   proceed(): void {
+    const message: string = this.validateInputs();
+    if (message) {
+      this.validationError(message);
+      this.stageState[this.stageNumber].verified = false;
+    } else {
+      // For stage 1 write a verified true to stage 1
+      this.stageState[this.stageNumber].verified = true;
+      this.next();
+    }
+  }
+
+  detached(): void {
```

Comment:
All these `detached` are identical.  they should in the base class.

Reply:
I thought of it but they call methods implemented in stage classes

Reply:
What methods?